src/main/java/com/example/bookshop/model/ShoppingCart.java

```diff
@@ -0,0 +1,38 @@
+package com.example.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE shopping_carts SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted=false")
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+
+    @OneToOne
+    @MapsId
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "shoppingCart")
```

Comment:
```suggestion
    @ToString.Exclude
    @EqualsAndHashCode.Exclude
    @OneToMany(mappedBy = "shoppingCart")
```

Reply:
FetchType `EAGER` in this situation it's not a good choice, try to avoid `LazyInitializationException` in a different way. e.g. `@Transactional(readOnly = true)` on methods.